src/ifcopenshell-python/ifcopenshell/file.py

```diff
@@ -81,11 +209,15 @@ def create_entity(self, type, *args, **kwargs):
             >>> #3=IfcPerson('Foobar',$,$,$,$,$,$,$)
         """
         e = entity_instance((self.schema, type))
+        # Hack, can a entity access its file?
```

Comment:
yeah we need something for this, probably implemented in a different way so that not only create_entity() but also attribute access and other ways of obtaining an entity_instance have a way to access their file